values/mainnet/network/api-read-slave-1.yaml

```diff
@@ -1,6 +1,6 @@
 nodeSelector:
   default:
-    nodeGroupName: group6
+    nodeGroupName: group8
```

Comment:
is it normal?

Reply:
Copy paste ninja
![image](https://user-images.githubusercontent.com/7295623/203985041-5c892981-b527-4eea-b5a7-8586bcd2036f.png)

Reply:
https://github.com/glifio/filecoin-chart/pull/95/files#diff-7a84d323a4d1c01aaaf65ede56e35de171cf6768b5a8ec6da75edc1a787fb0f5

Reply:
This is okay, please double check your code next time :)

Reply:
Thank you! I'll do this)